extensions/sql-database-projects/src/models/dataSources/sqlConnectionStringSource.ts

```diff
@@ -35,7 +35,21 @@ export class SqlConnectionDataSource extends DataSource {
 	}
 
 	public get integratedSecurity(): boolean {
-		return this.getSetting(constants.integratedSecuritySetting).toLowerCase() === 'true';
+		return this.getSetting(constants.integratedSecuritySetting)?.toLowerCase() === 'true';
+	}
+
+	public get azureMFA(): boolean {
+		return this.getSetting(constants.authenticationSetting)?.toLowerCase().includes(constants.activeDirectoryInteractive);
+	}
```

Comment:
I'm second guessing how I set this up originally since it allows for both multiple security schemes to be set... which would be invalid... Do you think that's something we should care/police for here, or should we just let the driver figure it out?

Reply:
I think it's ok to not care for now and expect the customer to provide a valid connection string. SSDT doesn't throw an error when a publish profile has a connection string with both integrated security=true and Authentication="Active Directory Interactive." I'll change the auth type to be AzureMFA if both are present to match SSDT.

Reply:
Kk.  I wasn't sure whether it would be a good idea for the `sqlConnectionStringSource` class to have an "authenticationType" property instead of multiple flags for `integratedSecurity` and `azureMFA`.  I think it'd make sense from the object perspective, but maybe not from the connection string side.  I'm still slightly on the fence... do you think it would cause issues if this object modeled the actual connection info rather than just the connection string?

Reply:
I'm not that concerned about it. Would it make more sense to have a helper function that converts a SqlConnectionDataSource to a ConnectionProfile and the authentication type can be determined there?